src/hooks.ts

```diff
@@ -65,7 +65,7 @@ export const useLoadOnScroll = (
       window.addEventListener('scroll', onscroll);
 
       // If the content is less than the screen height, fetch the next page.
-      const hasNoScroll = document.body.scrollHeight <= window.innerHeight;
+      const hasNoScroll = (document.body.scrollHeight - loadLimit) <= window.innerHeight;
```

Comment:
This was a small bug that if the last line was half filled, it was not calling a new fetch